NickvisionTagger.Shared/Controllers/MainWindowController.cs

```diff
@@ -115,7 +115,20 @@ public class MainWindowController
     public MainWindowController()
     {
         Aura = new Aura("org.nickvision.tagger", "Nickvision Tagger", _("Tagger"), _("Tag your music"));
+        if (Directory.Exists($"{Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData)}{Path.DirectorySeparatorChar}Nickvision{Path.DirectorySeparatorChar}{AppInfo.Name}"))
+        {
+            // Move or delete config files from older versions
```

Comment:
Could you add this updated version to Parabolic as well?

Reply:
Yes, I'm going to update Aura everywhere except Cavalier (I don't need Saved event there, but will need UserDirectories when I will implement them in Aura, so I'm goind to skip one version to avoid unnecessary work)

Reply:
Ok thank you ❤️
If you could update `Application` too just so we can keep it updated with new `ConfigurationBase`